app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Task;
+use App\Http\Requests\TaskRequest;
+use Validator, Input, Redirect;
+class TaskController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $tasks = Task::orderBy('created_at', 'asc')->get();
+
+        return view('tasks', [
+            'tasks' => $tasks
+        ]);
```

Comment:
Em nghiên cứu phần này nhé https://laravel.com/docs/5.5/views#passing-data-to-views